### PR TITLE
2.12.4

### PR DESCRIPTION
**Pull request by akravchenko564**

### CODE REVIEW

File: javascript.ru/2.12.4/2.12.4.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+		alert('Wellcome!');
+	}	
+	else if (password == 'Cancel') {
+		alert('Canceled');
+	}
+	else {
+		alert('Not correct password');
+	}
+}
+
```
Comment (KasichMax) [inline javascript.ru/2.12.4/2.12.4.js]: Remove this empty line. It is better not to insert line break between if and else.

File: javascript.ru/2.12.4/2.12.4.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	}
+	else {
+		alert('Not correct password');
+	}
+}
+ 
+else if (user == 'Cancel') {
+	alert('Login canceled');
+}
+
```
Comment (KasichMax) [inline javascript.ru/2.12.4/2.12.4.js]: Remove this empty line.

File: javascript.ru/2.12.4/2.12.4.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+﻿var user = prompt('Who are you?');
+
+if (user == 'Admin') {
```
Comment (KasichMax) [inline javascript.ru/2.12.4/2.12.4.js]: What about case insensitive check for all `if` except `if (password == 'Loki')`?

File: javascript.ru/2.12.4/2.12.4.js (hunk 2 above)
Comment (akravchenko564) [reply javascript.ru/2.12.4/2.12.4.js]: Done a9e11ec.

File: javascript.ru/2.12.4/2.12.4.js (hunk 1 above)
Comment (akravchenko564) [reply javascript.ru/2.12.4/2.12.4.js]: Done a9e11ec.

File: javascript.ru/2.12.4/2.12.4.js (hunk 3 above)
Comment (akravchenko564) [reply javascript.ru/2.12.4/2.12.4.js]: Done a9e11ec.

**Review by KasichMax — APPROVED** [review]

**Pull request merged**